apps/mobile/.env.preview

```
@@ -0,0 +1,25 @@
+# Preview Environment Variables (for internal testing)
+NODE_ENV=development
+API_BASE_URL=https://staging-api.wheresballthough.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_BASE_URL key should go before the NODE_ENV key

(UnorderedKey)